src/pages/Report.tsx

```diff
@@ -0,0 +1,213 @@
+                  </p>
+                  <ul className="list-disc ml-5 text-gray-600 text-sm">
+                    <li>seja curiosa por natureza;</li>
+                    <li>perfil mão na massa e ágil;</li>
+                    <li>sabe ouvir e se comunicar;</li>
+                    <li>busca e tenha um histórico de aprendizado contínuo;</li>
+                    <li>
+                      busca e fomenta a colaboração, mas também tenha as suas
+                      contribuições individuais inovadoras;
+                    </li>
+                    <li>
+                      entenda o objetivo e propósito da empresa, e consiga atuar
+                      com autonomia dentro de suas responsabilidades.
+                    </li>
+                  </ul>
+                  <p className="font-medium  text-gray-600 my-3">
+                    {" "}
+                    Responsabilidades e atribuições:
+                  </p>
+                  <p className="font-medium  text-gray-600 my-3">
+                    Culturalmente, procuramos uma pessoa que:
+                  </p>
+                  <ul className="list-disc ml-5 text-gray-600 text-sm">
+                    <li>seja curiosa por natureza;</li>
+                    <li>perfil mão na massa e ágil;</li>
+                    <li>sabe ouvir e se comunicar;</li>
+                    <li>busca e tenha um histórico de aprendizado contínuo;</li>
+                    <li>
+                      busca e fomenta a colaboração, mas também tenha as suas
+                      contribuições individuais inovadoras;
+                    </li>
+                    <li>
+                      entenda o objetivo e propósito da empresa, e consiga atuar
+                      com autonomia dentro de suas responsabilidades.
+                    </li>
+                  </ul>
+                  <p className="font-medium  text-gray-600 my-3">
+                    {" "}
+                    Responsabilidades e atribuições:
+                  </p>
+                  <p className="font-medium  text-gray-600 my-3">
+                    Culturalmente, procuramos uma pessoa que:
+                  </p>
+                  <ul className="list-disc ml-5 text-gray-600 text-sm">
+                    <li>seja curiosa por natureza;</li>
+                    <li>perfil mão na massa e ágil;</li>
+                    <li>sabe ouvir e se comunicar;</li>
+                    <li>busca e tenha um histórico de aprendizado contínuo;</li>
+                    <li>
+                      busca e fomenta a colaboração, mas também tenha as suas
+                      contribuições individuais inovadoras;
+                    </li>
+                    <li>
+                      entenda o objetivo e propósito da empresa, e consiga atuar
+                      com autonomia dentro de suas responsabilidades.
+                    </li>
+                  </ul>
+                  <p className="font-medium  text-gray-600 my-3">
```

Comment:
Remover espaçamento duplo entre as classes!
```suggestion
                  <p className="font-medium text-gray-600 my-3">
```